client/components/app.js

```diff
@@ -39,12 +46,20 @@ class App extends React.Component {
     }
   }
 
+  updateQuantityChoice(choice) {
+    this.setState({quantityChoice: Number(choice)});
+  }
+
+  updatePersonalizationChoice(choice) {
+    this.setState({personalizationChoice: choice});
+  }
+
   updateState(newData) {
     this.setState(newData);
   }
 
   componentDidMount() {
-    this.requestProductDetails(13)
+    this.requestProductDetails(11)
       .then(data => this.updateState(data));
   }

```

Comment:
You could handle the `.then` promise inside your `requestProductDetails` by replacing the `return response.data` line. It works either way though.